Install dir:
    /usr/libexec/lightdm-guest-session-wrapper
    /usr/libexec/lightdm-set-defaults
    /usr/libexec/lightdm/gdmflexiserver

Due to the following change, pkglibexec_SCRIPTS is invalid:
    http://git.savannah.gnu.org/cgit/automake.git/commit/?id=9ca632642b006ac6b0fc4ce0ae5b34023faa8cbf

---
 utils/Makefile.am |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/utils/Makefile.am b/utils/Makefile.am
index 41df48b..ee527cf 100644
--- a/utils/Makefile.am
+++ b/utils/Makefile.am
@@ -1,6 +1,7 @@
 bin_PROGRAMS = dm-tool
 libexec_PROGRAMS = lightdm-set-defaults
-dist_pkglibexec_SCRIPTS = gdmflexiserver
+ldmlibexecdir = $(libexecdir)/@PACKAGE@
+dist_ldmlibexec_SCRIPTS = gdmflexiserver
 
 dm_tool_SOURCES = \
 	dm-tool.c
